ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -14,24 +14,25 @@ cat << EOF > ~/inventory
 sshd.bastion-telco ansible_ssh_user=tester ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" ansible_ssh_private_key_file=/var/run/ssh-pass
 EOF
 
-# We will switch to kcli in the near future
 cat << EOF > ~/ocp-install.yml
 ---
-- name: Prepare to run ansible-ipi-install playbook
+- name: Grab and run kcli to install openshift cluster
   hosts: all
   tasks:
   - name: Clone repo
     git:
-      repo: https://github.com/openshift-kni/baremetal-deploy.git
-      dest: "~/baremetal-deploy"
+      repo: https://github.com/karamb/kcli-openshift4-baremetal.git
+      dest: "~/kcli-openshift4-baremetal"
       version: master
       force: yes
     retries: 5
+  - name: Remove old installer vm
+    shell: kcli delete vm --yes cnfdc5-installer
   - name: Run deployment
-    shell: ansible-playbook -i /home/tester/inventory playbook.yml
+    shell: kcli create plan --paramfile /home/tester/kcli_parameters.yml upstream_ci --wait
```

Comment:
a real use case scenario, will need to tweak the parameters file, 
at least to get the right version installed

Reply:
I believe we can pass things from the command line for this, right @karmab ? So we could remove something like the version from the kcli_parameters.yml file and pass it as an option on the cli

Reply:
yes, that's right

Reply:
also plan name, would probably be parameterized in the futuer if/when we have more envs
but that's fine